doc/source/release/1.21.0-notes.rst

```diff
@@ -359,7 +359,7 @@ Added a mypy plugin for handling platform-specific ``numpy.number`` precisions
 ------------------------------------------------------------------------------
 A mypy_ plugin is now available for automatically assigning the (platform-dependent)
 precisions of certain `~numpy.number` subclasses, including the likes of
-`~numpy.int_`, `~numpy.intp` and `~numpy.longlong`. See the documentation on
+`~numpy.long`, `~numpy.intp` and `~numpy.longlong`. See the documentation on
```

Comment:
I think this is misleading, as in 1.21 `np.long` meant something different.

Reply:
Ah right, I added this change by accident. Reverted!